apps/admin-gui-e2e/src/e2e/resource-observer-perun_admin.cy.js

```diff
@@ -17,13 +17,7 @@ context('Actions', () => {
   });
 
   beforeEach(() => {
-    cy.visit('organizations')
-      .get(`[data-cy=${dbVoName}]`)
-      .click()
-      .get('[data-cy=resources]')
-      .click()
-      .get('[data-cy=resource-list]')
-      .click()
+    cy.visit('home')
```

Comment:
I have just one suggestion. In several tests, we have `cy.visit('home').get('something')` in the `beforeEach()` section, but I would maybe prefer to use more concrete navigation, similar to what you deleted here. If we extend these tests in the future and the user will have more objects, we won't be able to be sure that the object that we want is shown between recently viewed objects. 
Another possible solution is to navigate to the object in this way at least in the `before()` section (then we will know that this object will be shown between recently viewed objects) and then use the shortened version in `beforeEach()` section.
What do you think about it @sarkapalkovicova @xflord ?

Reply:
Starting from 'home' and clicking through the side menu would probably safest and cover more of the app, no?
Also a side-note, the `cy.visit('organizations')` line is causing problems (at least that's what i deduced) in the e2e tests (see here: https://gitlab.ics.muni.cz/perun/e2e-perun/-/jobs/650843/artifacts/file/perun-rpc/cypress/videos/resource-self-service-perun_admin.cy.js.mp4). 
The test visits 'service-access' in the 'before' phase, starts 'beforeEach' where it's rerouted to 'organizations' but then a redirect to 'home' happens, which (I believe) is from the authorization flow.
 I remembered it now that you mentioned this and what Sarka did here would theoretically fix it.

Reply:
Good point, I agree that probably the best solution is to start from `home` and continue through the side menu. So we should avoid direct redirecting (like for example `cy.visit('organizations')`) in `beforeEach()` section and we should also avoid continuing from home through recently viewed objects (cards).